packages/dataviews/src/view-list.tsx

```diff
@@ -50,7 +50,7 @@ interface ListViewProps {
 	id: string;
 	isLoading: boolean;
 	onSelectionChange: ( selection: Item[] ) => void;
-	selection: Item[];
+	selection: string[];
```

Comment:
I think the selection objects holds just the ids of the items but it's weird that the `onSelectionChange` receives the whole objects. It's not consistent but it's the way it is for now.